File: src/main/java/tfg/proyecto/TFG/dtos/DTOeventoBajadaCarrito.java
package tfg.proyecto.TFG.dtos;

import java.time.LocalDateTime;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DTOeventoBajadaCarrito {
	private Long id;
	private String nombre;
	private String localizacion;
	private LocalDateTime inicioEvento;
	private LocalDateTime finEvento;
	private String imagen;
	private double precio;

}

File: appMovilShowPass/app/src/main/java/com/example/appmovilshowpass/data/remote/dto/DTOeventoBajadaCarrito.kt
package com.example.appmovilshowpass.data.remote.dto

import com.example.appmovilshowpass.model.Evento

data class DTOeventoBajadaCarrito(
    val id: Long,
    val nombre: String,
    val imagen: String,
    val precio: Double
)


fun DTOeventoBajadaCarrito.toEvento(): Evento {
    return Evento(
        id = id,
        nombre = nombre,
        localizacion = "",
        invitados = emptyList(),
        imagen = imagen,
        inicioEvento = "",
        finEvento = "",
        descripcion = "",
        carrusels = emptyList(),
        precio = precio,
        categoria = com.example.appmovilshowpass.model.Categoria.OTROS // o el default que uses
    )
}
